packages/app/src/store/studio-store.ts

```diff
@@ -324,16 +324,6 @@ export const useStudioStore = defineStore('studioRecorder', {
     visitUrl (url?: string) {
```

Comment:
Is `visitUrl` needed anymore? I suppose we can remove it when we remove all of the other studio code.